references/kvm-admin-api/apiproxy/policies/RF-404.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ Copyright 2021 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<RaiseFault async="false" name="RF-404">
+    <FaultResponse>
+        <Set>
+            <StatusCode>404</StatusCode>
+            <ReasonPhrase>Not Found</ReasonPhrase>
+            <Payload contentType="application/json">{"code": "404", "message": "Not Found"}</Payload>
```

Comment:
according to https://community.apigee.com/articles/23724/an-error-handling-pattern-for-apigee-proxies.html

it would be great if code also identified the proxy and error type.

since this will be an admin API, we could use `404.99.01` or similar